core/src/input.rs

```diff
@@ -67,66 +109,92 @@ impl InputManager {
         }
     }
 
-    pub fn map_input_event(&mut self, event: PlayerEvent) -> Option<PlayerEvent> {
-        // Optionally transform gamepad button events into key events.
-        match event {
+    pub fn process_input_event(&mut self, event: PlayerEvent) -> Option<InputEvent> {
```

Comment:
I feel like the name doesn't really make much sense now that we actually have an InputEvent type. It should probably be `process_player_input` or something like that.

Reply:
Changed to `process_event` to be as neutral as possible :switzerland: